src/datastore/models.py

```diff
@@ -1 +1,43 @@
-# Create your models here.
+from django.db import models
+
+
+class HistoricoAlerta(models.Model):
```

Comment:
@fccoelho please let me know if there are columns you think that should not be fetched from the infodengue's table

Reply:
All columns can be served